lib/input.js

```diff
@@ -97,6 +97,12 @@ function _createInputDescriptor (input, inputOptions, containerOptions) {
         inputDescriptor.rawWidth = inputOptions.raw.width;
         inputDescriptor.rawHeight = inputOptions.raw.height;
         inputDescriptor.rawChannels = inputOptions.raw.channels;
+
+        if (is.bool(inputOptions.raw.premultiplied)) {
+          inputDescriptor.rawPremultiplied = inputOptions.raw.premultiplied;
+        } else {
+          inputDescriptor.rawPremultiplied = false;
+        }
```

Comment:
This could also just be `inputDescriptor.rawPremultiplied = !!inputOptions.raw.premultiplied;` if that's preferable.

Reply:
Yes please, sounds good.